src/awsexplorer/regionNode.ts

```diff
@@ -26,54 +26,75 @@ import { DefaultS3Client } from '../shared/clients/s3Client'
 import { DefaultSchemaClient } from '../shared/clients/schemaClient'
 import { getEcsRootNode } from '../ecs/model'
 import { compareTreeItems, TreeShim } from '../shared/treeview/utils'
+import { Ec2ParentNode } from '../ec2/explorer/ec2ParentNode'
+import { DevSettings } from '../shared/settings'
+import { Ec2Client } from '../shared/clients/ec2Client'
 
 const serviceCandidates = [
     {
         serviceId: 'apigateway',
+        when: () => true,
         createFn: (regionCode: string, partitionId: string) => new ApiGatewayNode(partitionId, regionCode),
     },
     {
         serviceId: 'apprunner',
+        when: () => true,
         createFn: (regionCode: string) => new AppRunnerNode(regionCode, new DefaultAppRunnerClient(regionCode)),
     },
     {
         serviceId: 'cloudformation',
+        when: () => true,
         createFn: (regionCode: string) => new CloudFormationNode(regionCode),
     },
     {
         serviceId: 'logs',
+        when: () => true,
         createFn: (regionCode: string) => new CloudWatchLogsNode(regionCode),
     },
+    {
+        serviceId: 'ec2',
+        when: () => DevSettings.instance.isDevMode(),
+        createFn: (regionCode: string, partitionId: string) =>
+            new Ec2ParentNode(regionCode, partitionId, new Ec2Client(regionCode)),
+    },
     {
         serviceId: 'ecr',
+        when: () => true,
         createFn: (regionCode: string) => new EcrNode(new DefaultEcrClient(regionCode)),
     },
     {
         serviceId: 'ecs',
+        when: () => true,
```

Comment:
I was hoping this wouldn't be needed, that's what the `service.when === undefined` check is for.